paper/paper.md

```diff
@@ -152,34 +94,26 @@ using JSOSolvers
 max_time = 300. # run at most 5min
 callback = (LeNetNLPModel, solver, stats) -> FluxNLPModels.minibatch_next_train!(LeNetNLPModel)
 
-solver_stats = R2(LeNetNLPModel; callback, max_time)
-
-## Report on test data
+solver_stats = R2(LeNetNLPModel; callback, max_time) # We collect the status of Solver run
 test_accuracy = FluxNLPModels.accuracy(LeNetNLPModel)
 ```
 
-Another choice to train `LeNetNLPModel` is the LBFGS linesearch solver:
+Another choice to train `LeNetNLPModel` is the LBFGS solver with linesearch:
 ```julia
 solver_stats = lbfgs(LeNetNLPModel; callback, max_time)
-test_accuracy = FluxNLPModels.accuracy(LeNetNLPModel)
 ```
 
-To exploit any non-convexity present in `LeNetNLPModel`, an `LSR1` approximation of the Hessian can be employed and fed into the `trunk` solver.
-The trunk solver utilizes a trust-region method with a backtracking line search.
+To exploit any non-convexity present in `LeNetNLPModel`, an `LSR1` approximation of the Hessian which can be employed and fed into the `trunk` solver, utilizes a trust-region method with a backtracking linesearch.
 To integrate the LSR1 approximation and trunk into the training process, the code can be modified as:
 
 ```julia
-using NLPModelsModifiers # define also LSR1Model
+using NLPModelsModifiers # defines LSR1Model
 
 lsr1_LeNet = NLPModelsModifiers.LSR1Model(LeNetNLPModel)
 callback_lsr1 = 
   (lsr1_LeNet, solver, stats) -> FluxNLPModels.minibatch_next_train!(
                                                  lsr1_LeNet.model
                                                )
-
```

Comment:
```suggestion
solver_stats = trunk(lsr1_LeNet; callback = callback_lsr1, max_time)
```

Reply:
I do this manually, git gives me error

Reply:
What is the error?
It is related to the software engineer part or the optimization part?